src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.ts

```diff
@@ -18,25 +19,24 @@ import { ExerciseService } from 'app/exercises/shared/exercise/exercise.service'
 import { AssessmentType } from 'app/entities/assessment-type.model';
 import { ProgrammingExerciseStudentParticipation } from 'app/entities/participation/programming-exercise-student-participation.model';
 import { ProgrammingExercise } from 'app/entities/programming-exercise.model';
-import { SubmissionExerciseType } from 'app/entities/submission.model';
 import { formatTeamAsSearchResult } from 'app/exercises/shared/team/team.utils';
 import { AccountService } from 'app/core/auth/account.service';
 import { setBuildPlanUrlForProgrammingParticipations } from 'app/exercises/shared/participation/participation.utils';
 import { faCodeBranch, faDownload, faFolderOpen, faListAlt, faSync } from '@fortawesome/free-solid-svg-icons';
 import { faFileCode } from '@fortawesome/free-regular-svg-icons';
 import { Range } from 'app/shared/util/utils';
+import dayjs from 'dayjs/esm';
 import { ExerciseCacheService } from 'app/exercises/shared/exercise/exercise-cache.service';
 
 /**
  * Filter properties for a result
  */
-enum FilterProp {
+export enum FilterProp {
     ALL = 'all',
-    SUCCESSFUL = 'successful',
-    UNSUCCESSFUL = 'unsuccessful',
```

Comment:
Can we keep the successful / unsuccessful filters? I use them regularly.

Reply:
Readded, now as dropdown for all filters